collector/mongod/index_counters.go

```diff
@@ -23,7 +23,7 @@ var (
 
 //IndexCounterStats index counter stats
 type IndexCounterStats struct {
-	Accesses  float64 `bson:"accesses`
+	Accesses  float64 `bson:"accesses"`
```

Comment:
Not entirely sure of the inner workings here but I think it would not match the bson output and therefore always report accesses as zero, but I am not 100% sure. Structs seem very forgiving of missing/undefined fields